pandas/_libs/tslibs/conversion.pyx

```diff
@@ -463,7 +530,7 @@ cdef _TSObject convert_str_to_tsobject(object ts, object tz, object unit,
                     if tz is not None:
                         # shift for localize_tso
                         ts = tz_localize_to_utc(np.array([ts], dtype='i8'), tz,
-                                                ambiguous='raise')[0]
+                                                ambiguous=not(fold))[0]
```

Comment:
Dont think we need the parenthesis here.

Reply:
Done. An old habit sometimes creeps up on me.